_scripts/instruction-widget/install.js

```diff
@@ -154,6 +166,8 @@ module.exports = function(context) {
 
     if (context.webserver == "apache") {
       context.package = "python-certbot-apache";
+    } else if (context.webserver == "nginx") {
+      context.certonly = true;
```

Comment:
The maintainer has opened a review request.

btw, I'm keeping track of all of these here: https://docs.google.com/spreadsheets/d/1cudoHzxjl1TEUbV7OCDW1bXcivzLnS2zGzqEnehJFq0/edit?usp=sharing